env.d.ts

```diff
@@ -14,6 +14,15 @@ type ScriptFailure = {
 type ScriptResponse<T = object> = ScriptSuccess<T> | ScriptFailure
 type ErrorScripts = Record<string, () => ScriptFailure>
 
+type Scriptor<Args = unknown, Ret = ScriptResponse> = Pick<Function, "name" | "call"> ;
```

Comment:
a scriptor is not a `Function`. it happens to also have `.name` property and `.call()` method, but it's not a `Function`